### PR TITLE
[#5] [UI] As a user, I can see a Progress HUD

### PR DESCRIPTION
**Pull request by markgravity**
https://github.com/markgravity/kmm-ic/issues/5

## What happened 👀

- Add Progress HUD UI

## Insight 📝

Please check `ProgressHUD_Previews` to have a better insight into the final result

## Proof Of Work 📹

CleanShot 2023-04-13 at 11 19 02

### CODE REVIEW

**Review by phongvhd93 — APPROVED** [review]

File: .github/workflows/deploy_production_appstore.yml [hunk 1]
```diff
@@ -0,0 +1,99 @@
+name: Deploy Staging Build To Firebase
```
Comment (suho) [inline .github/workflows/deploy_production_appstore.yml]: Why UI tasks but have 2 new workflows?

File: ios/KMMIC/Resources/Assets/Assets.xcassets/AppBackground.imageset/AppBackground.png
Comment (suho) [inline ios/KMMIC/Resources/Assets/Assets.xcassets/AppBackground.imageset/AppBackground.png:1]: Same with these new files, it's not related to the scope of the task

**Review by suho — CHANGES_REQUESTED** [review]

File: .github/workflows/deploy_production_appstore.yml (hunk 1 above)
Comment (markgravity) [reply .github/workflows/deploy_production_appstore.yml]: @suho I rebased the branch 🙏

File: ios/KMMIC/Resources/Assets/Assets.xcassets/AppBackground.imageset/AppBackground.png
Comment (markgravity) [reply ios/KMMIC/Resources/Assets/Assets.xcassets/AppBackground.imageset/AppBackground.png:1]: @suho I rebased the branch 🙏

**Review by suho — APPROVED** [review]

**Pull request merged**